### DISCUSSION TITLE
How do I get color mode in theme shadows?

### DISCUSSION THREAD

**Discussion by xddz9** [chakra-ui/chakra-ui#2787] [Q&A]
I'd like to change my theme's shadows and use different colors depending on whether color mode is dark or light. When extending the theme most fragments take an optional props variable that contains the current color mode, but that's not the case with shadows. I can think of a couple of alternatives but they all feel hacky, so is there a recommended way of doing that?

Thanks in advance!

**Comment by la55u** [comment]
Related: https://github.com/chakra-ui/chakra-ui/discussions/2738
Unfortunately I haven't found a nice way to do it either
